### PR TITLE
eas deployment and auth persistence changes

### PR DESCRIPTION
**Pull request by Manan-Sharma-5**
## Summary by CodeRabbit

- **New Features**
	- Updated the app's URL scheme to "trendly-creators."
	- Enhanced user tracking and session management with new permissions handling and Facebook SDK integration.
	- Introduced Facebook login functionality in the PreSignIn component.
- **Bug Fixes**
	- Maintained error handling for Facebook login failures.
- **Chores**
	- Added new dependencies for tracking transparency and Facebook SDK integration.
	- Added a new script for build pre-installation.

### CODE REVIEW

**Comment by Manan-Sharma-5** [comment]
[Facebook Authentication is having issues in the native (android)](https://trello.com/c/QFLReh9T/95-facebook-authentication-is-having-issues-in-the-native-android)

**Comment by Manan-Sharma-5** [comment]
[Figure out what issue is coming in the eas build.](https://trello.com/c/847DXLMP/31-figure-out-what-issue-is-coming-in-the-eas-build)

**Comment by Manan-Sharma-5** [comment]
[None of the page were loading -> Continous loader were showing on clicking any of the tabs (mostly firestore link was broken)](https://trello.com/c/G2uTmeEu/32-none-of-the-page-were-loading-continous-loader-were-showing-on-clicking-any-of-the-tabs-mostly-firestore-link-was-broken)

File: contexts/auth-context.provider.tsx [hunk 1]
```diff
@@ -62,6 +52,7 @@ export const AuthContextProvider: React.FC<PropsWithChildren> = ({
   const [[isLoading, session], setSession] = useStorageState("id");
   const [user, setUser] = useState<User | null>(null);
   const router = useRouter();
+  console.log("Session: ", session);
```
Comment (dev-rathore) [inline contexts/auth-context.provider.tsx:55]: Lets remove all the consoles everywhere.

**Review by dev-rathore — APPROVED** [review]

**Pull request merged**